src/thor/route_action.cc

```diff
@@ -175,6 +175,9 @@ std::string thor_worker_t::expansion(Api& request) {
                                 baldr::GraphId edgeid, const char* status, bool full_shape = false) {
     // full shape might be overkill but meh, its trace
     auto tile = reader.GetGraphTile(edgeid);
+    if (tile == nullptr) {
+      return;
```

Comment:
i'm not so sure about what to do wit this one. the point of the expansion API is to visualize the way the algorithm traverses the graph. if it gets here that means it did in the algorithm but we cant track it properly because the tile disappeared... its this kind of thing that makes me think this is just a terrible idea in general.

i get the idea of "best attempt" but the scenario in which that is an acceptable outcome is very limited.

anyway i think we should at least `LOG_ERROR` here that the expansion result will be incomplete